src/diffParser.js

```diff
@@ -0,0 +1,37 @@
+const diffParser = () => {
+  let diffString;
+  let isReading = false;
+
+  const possibleFiles = [
+    '.yml',
+    '.yaml',
+    '.md',
+    '.json',
+    '.lock',
+    '.git',
+    '.cjs',
+    '.config.js',
+    'LICENSE',
+  ];
+
+  const checkFile = (line) => {
+    let invalidFiles = 0;
+    possibleFiles.forEach((ext) => {
+      invalidFiles += line.includes(ext);
+    });
+    return Boolean(invalidFiles);
+  };
+
+  diff.split(/\r?\n/).forEach((line) => {
+    if (line.slice(0, 5) === '+++ b' && checkFile(line)) {
+      isReading = false;
+    } else if (line.slice(0, 5) === '+++ b' && !checkFile(line)) {
+      isReading = true;
+    }
+
+    if (line[0] === '+' && isReading) {
+      diffString += line;
+    }
+  });
```

Comment:
I would also break this code into smaller functions.

The code should answer these questions (without coments)
- What does the split do?
- What does the first if is about?
- What does the else if (the one with the slice) do?

Also, `checkFile` is not a good name. What are you checking on the file?